pkgs/servers/sql/postgresql/packages.nix

```diff
@@ -44,6 +44,8 @@ self: super: {
 
     pgsql-http = super.callPackage ./ext/pgsql-http.nix { };
 
+    pgvecto-rs = super.callPackage ./ext/pgvecto-rs { };
+
```

Comment:
@infinisil by-name CI fails here but I don't think it should since this isn't all-packages.nix.

Reply:
It only started failing after I edited the PR description last night btw
I'll also take a look to see if it's something I can fix on my side, but it seems to be a problem with the CI at first glance

Reply:
Seems to have been caused by commit https://github.com/NixOS/nixpkgs/commit/a8a7369c36120acb9ca43376504b6e9babed7261, which was reverted in the meantime, https://github.com/NixOS/nixpkgs/commit/ac637ef21ecc2f0a1a0987fe9ce1d7f0d7f42fca.
I've edited the PR description again to trigger the CI and it passes now.